adapters/connatix/connatix.go

```diff
@@ -199,7 +199,8 @@ func buildRequestImp(imp *openrtb2.Imp, ext impExtIncoming, displayManagerVer st
 
 	impExt := impExt{
 		Connatix: impExtConnatix{
-			PlacementId: ext.Bidder.PlacementId,
+			PlacementId:           ext.Bidder.PlacementId,
+			ViewabilityPercentage: ext.Bidder.ViewabilityPercentage,
```

Comment:
If `ViewabilityPercentage`'s value should range from 0 to 1, should we place a validation somewhere? If not in `static/bidder-params/connatix.json`, here just before assigning?

Reply:
I don’t want to enforce this because is handled on our side and should not block the request.